docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -77,89 +51,20 @@ Open two windows, which we'll refer to throughout this walkthrough as:
+- create the ManagedClusterSet that uses the ingress label to select clusters
+- bind this cluster set to our multi-cluster-gateways namespace so that we can use those clusters to place gateways on
+- set up a placement resource, in order to place our gateways onto clusters
+- lastly, it will set up a multi-cluster gateway class
+
 1. Once this is completed your kubeconfig context should be set to the hub cluster. 
 
     > **Optional Step:** :thought_balloon: If you need to reset this run the following in `T1`:
     > ```bash
     > kind export kubeconfig --name=mgc-control-plane --kubeconfig=$(pwd)/local/kube/control-plane.yaml && export KUBECONFIG=$(pwd)/local/kube/control-plane.yaml
     > ```
-
-1. In the hub cluster (`T1`) we are going to label the control plane managed cluster as an Ingress cluster:
-
-    ```bash
-    kubectl label managedcluster kind-mgc-control-plane ingress-cluster=true
-    ```
-
-1. Next, in `T1`, create the ManagedClusterSet that uses the ingress label to select clusters:
-
-    ```bash
-    kubectl apply -f - <<EOF
-    apiVersion: cluster.open-cluster-management.io/v1beta2
-    kind: ManagedClusterSet
-    metadata:
-      name: gateway-clusters
-    spec:
-      clusterSelector:
-        labelSelector: 
-          matchLabels:
-            ingress-cluster: "true"
-        selectorType: LabelSelector
-    EOF
-    ```    
-
-1. Next, in `T1` we need to bind this cluster set to our multi-cluster-gateways namespace so that we can use those clusters to place gateways on:
-
-    ```bash
-    kubectl apply -f - <<EOF
-    apiVersion: cluster.open-cluster-management.io/v1beta2
-    kind: ManagedClusterSetBinding
-    metadata:
-      name: gateway-clusters
-      namespace: multi-cluster-gateways
-    spec:
-      clusterSet: gateway-clusters
-    EOF
-    ```
-
-### Create a placement for our gateways
-
-1. In order to place our gateways onto clusters, we need to setup a placement resource. Again, in `T1`, run:
-
-    ```bash
-    kubectl apply -f - <<EOF
-    apiVersion: cluster.open-cluster-management.io/v1beta1
-    kind: Placement
-    metadata:
-      name: http-gateway
-      namespace: multi-cluster-gateways
-    spec:
-      numberOfClusters: 1
-      clusterSets:
-        - gateway-clusters
-    EOF
-    ```    
-
-### Create the gateway class
- 
-1. Lastly, we will set up our multi-cluster gateway class. In `T1`, run:
-
-    ```bash
-    kubectl create -f hack/ocm/gatewayclass.yaml
-    ```
-
-### Start the Gateway Controller
-
-1. In `T1` run the following to build and deploy the Gateway Controller in a container:
-
-    ```bash
-    make build-controller kind-load-controller deploy-controller
-    ```
-
-    *Alternatively, in a new window we'll call `T3`, you can run the controller locally instead by running:*
-
-    ```bash
-    (export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-controller install run-controller)
-    ```
+   
```

Comment:
Nice! Probably not within the scope of this PR, but would it be helpful to have a summary of the clusters/resources that have been created up to this point here? Maybe a diagram too? Both of these could potentially be reused on the website.

Reply:
There is a list of what we are doing with the script added. if there is a suitable diagram somewhere, I'm happy to add that too.